lab-teddy/lib/bstNode.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+
+const bstNode = module.exports = function(val){
+  this.val = val;
+  this.data = null;
+  this.parent = null;
+  this.left = null;
+  this.right = null;
+};
+
+//O(n)
```

Comment:
So your Big-O notations aren't exactly correct. Instead of O(n) what else could this be?